### PR TITLE
Add support to vulnerability alert and automated security fixes APIs

### PR DESCRIPTION
**Pull request by cuichenli**
- Add functions to get vulnerability alert information and change the vulnerability alert settings. 
- Add functions to change the automated security fixes settings. 

Unit tests cover:
- Normal case 
- when the repository does not exist (error 404)
- change automated security fix when vulnerability alert is disabled(error 424)
- get vulnerability when it is turned off(error 404).

Note: Those APIs are currently in preview status, feel free to reject it. 

Github API Documents:
- [get vulnerability settings](https://developer.github.com/v3/repos/#check-if-vulnerability-alerts-are-enabled-for-a-repository)
- [enable vulnerability alerts](https://developer.github.com/v3/repos/#enable-vulnerability-alerts)
- [disable vulnerability alerts](https://developer.github.com/v3/repos/#disable-vulnerability-alerts)
- [enable automated security fixes](https://developer.github.com/v3/repos/#enable-automated-security-fixes)
- [disable automated security fixes](https://developer.github.com/v3/repos/#disable-automated-security-fixes)

### CODE REVIEW

File: tests/Repository.py [hunk 1]
```diff
@@ -693,3 +693,47 @@ def testOwner(self):
         owner = lazy_repo.owner
         eager_repo = self.getEagerRepository()
         self.assertEqual(owner, eager_repo.owner)
+
+    def testEnableVulnerabilityAlert(self):
+        lazy_repo = self.getLazyRepository()
+        self.assertEqual(lazy_repo.enable_vulnerability_alert(), True)
```
Comment (s-t-e-v-e-n-k) [inline tests/Repository.py]: All of these can use either self.assertTrue() and self.assertFalse() to simply the assertions.

**Comment by s-t-e-v-e-n-k** [comment]
Also, I'm not certain why codecov is so unhappy with this PR. (I have one that is exhibiting the same issue)

File: tests/Repository.py (hunk 1 above)
Comment (cuichenli) [reply tests/Repository.py]: Updated in the new commit.

**Comment by s-t-e-v-e-n-k** [comment]
Thanks, this looks great!

**Comment by sfdye** [comment]
@s-t-e-v-e-n-k I will pass you the release details

**Comment by sfdye** [comment]
@s-t-e-v-e-n-k Hey Steve, I've added the release docs to `DEPLOY.md`. Do you want to give a shot :)

**Pull request merged**